recipe_Menager.py

```diff
@@ -2,26 +2,105 @@ class RecipeManager:
     def __init__(self):
         self.recipes = []
 
-    @property
     def display_recipes(self):
         for recipe in self.recipes:
-            print(recipe)
-            return self.recipes
-            
-    @display_recipes.getter
-    def display_recipes(self):
-      return str(self.recipes)
+            print(recipe._title) #title instead the name 
```

Comment:
What is important to remember is how these classes are going to be used. Apart from displaying for debug, test and during development all the displaying will be done via a gui interface and not via the command line. Although its helpful to have these things its probably better to put them either in a test file or in another test class which is derived from this class.